docs/elasticsearch-spec.asciidoc

```diff
@@ -100,3 +100,30 @@ Example to create a Kubernetes TLS secret with a self-signed certificate:
 $ openssl req -x509 -newkey rsa:4096 -keyout tls.key -out tls.crt -days 365 -nodes
 $ kubectl create secret tls my-cert --cert tls.crt --key tls.key
 ----
+
+[id="{p}-virtual-memory"]
+=== Virtual memory
+
+By default, Elasticsearch is using memory mapping (mmap) to efficiently access indexes. Usually, default values for virtual address space on Linux distributions is too low for Elasticsearch to work properly, which may result in out of memory exceptions. To increase virtual memory ECK will set recommended value by default. This is requires the kubelet to allow running privileged containers.
```

Comment:
`default value is` or `default values are`

Reply:
`s|ECK will set recommended|ECK sets the recommended|`

`This requires that the kubelet is allowed to run privileged containers.`*

Reply:
Fixed